parse.py

```diff
@@ -271,6 +271,43 @@ def date_convert(
     return d
 
 
+dt_format_to_regex = {symbol: "[0-9]{2}" for symbol in "ymdIMSUW"}
+dt_format_to_regex.update({"-" + symbol: "[0-9]{1,2}" for symbol in "ymdIMS"})
+
+dt_format_to_regex.update(
+    {
+        "a": "(?:Sun|Mon|Tue|Wed|Thu|Fri|Sat)",
+        "A": "(?:Sunday|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday)",
+        "Y": "[0-9]{4}",
+        "H": "[0-9]{1,2}",
+        "B": "(?:January|February|March|April|May|June|July|August|September|October|November|December)",
+        "b": "(?:Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)",
```

Comment:
I wonder if these should be made locale-aware...

Reply:
yeah good idea I'll look into it

Reply:
Can be addressed at same time as https://github.com/r1chardj0n3s/parse/issues/1